src/dashboard/tests/test_backlog.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
```

Comment:
I used one of the other test scripts in that directory as a template and kept the utf-8 header and `from __future__ import absolute_import` from the other file, figuring it couldn't hurt with 2/3 compatibility. 

I just checked and these are already in elasticsearchFunctions, so we should be good there. Thanks for the tip on pylint!

Reply:
A lesson for me in clarity, e.g. elasticsearchFunctions, or any others, e.g. `src/MCPClient/lib/clientScripts/move_to_backlog.py` etc. :wink: but since you're not specifically thinking about `py3` compatibility, that's a-okay. This would normally be out of scope of a CR review I feel (again unless there was a clear motivation). Nothing to do here. It'll get picked up when the Client scripts are migrated.

Reply:
You're kind but I'm going to go ahead and blame the reader (me) for this misunderstanding :smile: